Configuration/src/Steeltoe.Samples.Configuration/manifest.yml

```diff
@@ -0,0 +1,17 @@
+---
+applications:
+- name: steeltoe-configuration-sample
+  buildpacks:
+   - dotnet_core_buildpack
+  memory: 128M
+  random-route: true
+  stack: cflinuxfs4

```

Comment:
Do we need `stack`? It's not in the connector manifest files.

Reply:
It really only comes into play if you cf push to Windows and then to Linux, but it shouldn't hurt to include it

Reply:
Good to know. That probably explains why I was having these issues with deployments of Connectors. Should it be added to connectors?

Reply:
Yeah, we should probably at this to connectors at some point